### ISSUE TITLE
Cannot upgrade to lumen 6.0.*

### ISSUE THREAD

**Issue by justasSendrauskas** [DarkaOnLine/SwaggerLume#77]
When trying to upgrade lumen from 5.8 to 6, swagger lume prevents it as has a dependency on previous versions:

```
- darkaonline/swagger-lume 5.6.0 requires laravel/lumen-framework ~5.6 -> satisfiable by laravel/lumen-framework[5.6.x-dev, 5.7.x-dev, 5.8.x-dev].
```

**Comment by DarkaOnLine** [comment]
6.0 is released with Lumen 6.0 support: https://github.com/DarkaOnLine/SwaggerLume/releases/tag/6.0

**Comment by justasSendrauskas** [comment]
brilliant, thank you very much;]
